pilot/pkg/serviceregistry/serviceentry/servicediscovery.go

```diff
@@ -131,8 +131,8 @@ func (s *ServiceEntryStore) workloadEntryHandler(old, curr model.Config, event m
 		return
 	}
 	log.Debugf("Handle event %s for workload entry %s in namespace %s", event, curr.Name, curr.Namespace)
-	instancesUpdated := []*model.ServiceInstance{}
-	instancesDeleted := []*model.ServiceInstance{}
+	instancesUpdated := make([]*model.ServiceInstance, 0)
+	instancesDeleted := make([]*model.ServiceInstance, 0)
```

Comment:
I think there is no diff here

Reply:
Yeah just being consistent by setting len(0). Will set it back to original to avoid any changes.